docs/config/README.md

```diff
@@ -745,7 +755,7 @@ error_symbol = '[➜](bold red) '
 # ~/.config/starship.toml
 
 [character]
-vicmd_symbol = '[V](bold green) '
+vimcmd_symbol = '[V](bold green) '
```

Comment:
I think you added some extra changes.

Reply:
Sorry, some things stuck over from a merge, reverted those back just now